src/docs/product/performance/transaction-summary.mdx

```diff
@@ -94,6 +105,31 @@ The **Tags** tab displays a list of suspect tag keys that often correspond to sl
 
 The table in **All Events** tab offers a full list of transactions broken down similar to the abbreviated table in the overview. Filters are carried across from other views. You can also narrow the events down by percentile.
 
+## Spans
+
+<Note>
+
+This feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony.
+If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.
+
+</Note>
+
+The "Spans" tab displays a list of suspect spans that correspond to where most of the time in a transaction is being spent and can help you find spans where performance problems lie. Choose from one of several [metrics](#metrics) to sort spans and identify different types of problems. Filter spans to only see the span operation type you're interested in. When you click an "Example Transaction" event, it takes you directly to the span in question.
+
+Suspect spans are determined using a span's exclusive time rather than its duration. To calculate a span's exclusive time, take the interval of the span and subtract the intervals of any child spans. The duration of the remaining interval is the span's exclusive time.
```

Comment:
I think it would be useful here to spell out the exclusive time of the span in the image below. Something like "In the example span shown below, the exclusive time is 25 ms."

Reply:
Good idea 👍